VastClient/VastClient/VastTracker.swift

```diff
@@ -60,11 +68,11 @@ public class VastTracker {
         self.trackingStatus = .tracking
         self.delegate = delegate
         self.totalAds = self.vastAds.count
-        
-        // FIXME: this will not work correctly for convenience init and also, ad break for VMAP with multiple ad breaks and time offsets will also not work as expected
-        delegate?.adBreakStart(vastTracker: self)
+        self.trackProgressCumulatively = trackProgressCumulatively
     }
     
+    // TODO: this should be removed in the future. Please, store the VMAPModel elsewhere and initialize this class with only one VastModel
+    @available(*, message: "Use init(id:,vastModel:) instead")
     public convenience init(id: String, vmapModel: VMAPModel, breakId: String, startTime: Double, supportAdBuffets: Bool = false, delegate: VastTrackerDelegate? = nil) throws {
```

Comment:
VMAP can contain beacons that are not present in VAST. We should still have this tracking class fire those beacons automatically.

Reply:
Ok, i agree.